server/daemon.py

```diff
@@ -262,9 +262,9 @@ def processor(result):
         network_info = await self.getnetworkinfo()
         return network_info['relayfee']
 
-    async def getrawtransaction(self, hex_hash):
+    async def getrawtransaction(self, hex_hash, verbose=False):
         '''Return the serialized raw transaction with the given hash.'''
-        return await self._send_single('getrawtransaction', (hex_hash, 0))
+        return await self._send_single('getrawtransaction', (hex_hash, verbose))
```

Comment:
we want int(verbose) because as the Bitcoin Core docs indicate, the integer form is more backwards-compatible than the boolean

return await self._send_single('getrawtransaction', (hex_hash, int(verbose)))

Reply:
Mhm, I so have to better understand the code duplication here, I thought this method was invoked by the one in the Controller, and so the verbose param is casted as int in the previous class and passed here always as an int.
However, it's not an issue to do again it, I guess.